static/js/components/EmbedlyCard.js

```diff
@@ -0,0 +1,36 @@
+/* global SETTINGS: false */
+// @flow
+import React from "react"
+import isURL from "validator/lib/isURL"
+
+import { loadEmbedlyPlatform } from "../lib/embed"
+import ReactDOMServer from "react-dom/server"
+
+type Props = {
+  url: string
+}
+export default class EmbedlyCard extends React.Component<Props> {
```

Comment:
if you wanted, this could just be a stateless component. `loadEmbedlyPlatform` is idempotent (if the thing is already there it just won't do anything) so it would be safe to just call it on every render of the component.

Reply:
My main concern is that the React docs recommend putting this kind of DOM manipulation in `componentDidMount` and leaving it out of `render` for clarity's sake.